packages/core-utils/src/query.js

```diff
@@ -284,6 +282,29 @@ export async function planParamsToQuery(params, config) {
         break;
       }
     }
+  });
+  return query;
+}
+
+/**
+ * Async method to create a otp query based on a the url params. This provides
+ * the same functionality as planParamsToQuery, except that it will also attempt
+ * to geocode the input from and to strings if no lat/lng values were provided.
+ *
+ * @param  {Object} params An object representing the parsed querystring of url
+ *    params.
+ * @param config the config in the otp-rr store.
+ */
+export async function planParamsToQueryAsync(params, config = {}) {
```

Comment:
Propose to rename this `planParamsToQueryWithPlaceGeocoding` to indicate it'll do geocoding.